py/client-ticking/setup.py

```diff
@@ -1,21 +1,39 @@
 #
 #  Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
 #
+import os
 import pathlib
+
+# Note: pkg_resources is deprecated https://setuptools.pypa.io/en/latest/pkg_resources.html, and it is suggested
+# to use an external library `packaging`. From the context of building a wheel though, we'd prefer to not have to
+# install extra dependencies, at least until we can more properly manage the build environment (pyproject.toml)
+from pkg_resources import parse_version
 from setuptools import find_packages, setup, Extension
 from Cython.Build import cythonize
 
-# The directory containing this file
-HERE = pathlib.Path(__file__).parent
+def _get_readme() -> str:
+    # The directory containing this file
+    HERE = pathlib.Path(__file__).parent
+    # The text of the README file
+    return (HERE / "README.md").read_text()
```

Comment:
If I'm not mistaken, you should specify utf8 encoding here: https://github.com/pypa/sampleproject/commit/aeeb50a948addcb712ad4261df472263514991e1#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L15

```suggestion
    return (HERE / "README.md").read_text(encoding="utf-8")
```
It probably defaults to utf-8 on most platforms, but I believe it should be explicit to be safe in case the platform locale is different.